packages/js/test/plugins/auctionHouseModule/withdrawFromTreasuryAccount.test.ts

```diff
@@ -0,0 +1,73 @@
+import test, { Test } from 'tape';
+import { sol } from '@/types';
+import {
+  metaplex,
+  killStuckProcess,
+  createNft,
+  createWallet,
+} from '../../helpers';
+import { createAuctionHouse } from './helpers';
+
+killStuckProcess();
+
+test('[auctionHouseModule] withdraw from treasury account on an Auction House', async (t: Test) => {
```

Comment:
A note on that test.

I think it's great that it is testing the royalty system but I think 80% of this test belongs to the "execute sale" operation.

Here we should get straight to the point and airdrop the treasury account so we can test the withdraw logic (just like we did for the `withdrawFromFeeAccount` operation).